sql/driver/driver_test.go

```diff
@@ -227,3 +227,60 @@ func TestInsecure(t *testing.T) {
 		t.Fatal(err)
 	}
 }
+
+func TestTransactions(t *testing.T) {
```

Comment:
If you implement `BEGIN`, `COMMIT`, `ROLLBACK`, you'll be able to use the logic-test framework instead of writing the test like this.